osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -22,6 +22,18 @@ public class DrawableCarouselBeatmapSet : DrawableCarouselItem, IHasContextMenu
     {
         public const float HEIGHT = MAX_HEIGHT;
 
+        public static float YLerp(float targetY, float currentY, double elapsed)
```

Comment:
Not a huge fan of having this static and called from outside. I think it's be more elegant if it could be non-static and update the panel itself, rather than return a value for various different use cases.

ie. it seems both use cases in `BeatmapCarousel` are operating  on the panel itself, so potentially all of that logic could be moved internal to this class (or the corresponding model) instead?

Reply:
> ie. it seems both use cases in ``BeatmapCarousel`` are operating on the panel itself

That turns out to actually not be the case. In ``Update()`` it's giving an initial position to the drawable, but in ``updateYPositions()`` it's operating on the ``CarouselBeatmapSet`` and not the drawable. 

The reason that I think this should be one function is exactly that items and their drawable representations need to be in-sync so that no weird edge cases can happen. If a copy of this function is moved both to ``CarouselBeatmapSet`` and ``DrawableCarouselBeatmapSet`` there's no guarantee somebody won't change one function and forget to change the other.

Reply:
if so, it should be in a separate helper class somewhere, or in the model.

Reply:
Can the function stay static in the helper class? What would even be a good name for such a class, ``CarouselHelper`` or ``CarouselUtils`` or something?

Reply:
it should be even probably a static class entirely. as for the name, either works for me, really.